src/main/java/duke/taskmanagement/Ui.java

```diff
@@ -0,0 +1,102 @@
+package duke.taskmanagement;
+import java.util.List;
+import java.util.Scanner;
+
+public class Ui {
+    public Scanner sc = new Scanner(System.in);
+
+    /**
+     * Constructor to create a Ui object.
+     */
+    public Ui() {
+    }
+
+    /**
+     * Returns the String that the scanner read in from user inputs.
+     * @return User input as string.
+     */
+    public String readInCmd() {
+            return sc.nextLine();
+    }
+
+    /**
+     * To print the greeting to user when running this program.
+     */
+    public void greet(){
+        System.out.println("Hello! I'm JJ\n" +
+                "What can I do for you?\n" + "\n");
+    }
+
+    /**
+     * To say goodbye to the user when the program ends.
+     */
+    public void bye(){
+        System.out.println("Bye. Hope to see you again soon!\n");
+    }
+
+    /**
+     * To print every element in the list.
+     * @param size The size of the list you wish to print.
+     * @param ls The List<Task> you wish to print.
+     */
+    public void printList(int size, List<Task> ls){
+        System.out.println("Here are the tasks in your list: ");
+        for(int i = 0; i < size;i++) {
```

Comment:
Perhaps a white space after the for keyword is better?
Perhaps semicolons in for statements followed by a space character are better?